pkgs/applications/editors/vscode/generic.nix

```diff
@@ -14,7 +15,7 @@
 
 let
   inherit (stdenv.hostPlatform) system;
-  unwrapped = stdenv.mkDerivation {
+  unwrapped = stdenv.mkDerivation rec {
```

Comment:
Some people say that `rec` is an antipattern. I'm not sure what the nixpkgs consensus is.

Reply:
I needed to reference buildInputs. I could deitehr factor it outside the attrset, or just add rec

Reply:
`rec` when you don't need it is bad (or at least I warn for it nix-linter), but I don't think I've heard anyone say that `rec` in general is an antipattern.

Reply:
It's been mentioned here: https://nix.dev/anti-patterns/language#rec-expression

Reply:
Using it libs and pure functions can lead to confusion but in mkDerivation it is totally normal to use and encouraged.